bentoml/_internal/yatai_client/config.py

```diff
@@ -0,0 +1,82 @@
+import os
+from pathlib import Path
+from typing import List
+
+import attr
+import cattr
+import yaml
+
+from bentoml._internal.yatai_client.yatai import YataiClient
+from bentoml._internal.yatai_client.errors import YataiCliError
+
+default_context_name = 'default'
+
+
+def get_config_path() -> Path:
+    return Path.home() / '.yatai.yaml'
```

Comment:
could we put this file under BENTOML_HOME? https://github.com/bentoml/BentoML/blob/cc111ce/bentoml/_internal/configuration/containers.py#L31